Traceback (most recent call last):
  File "/home/at-sac/yuzhu/qu5/qu5_pvm.py", line 254, in <module>
    #for i in range(1):
IndexError: list index out of range
srun: error: margpu010: task 0: Exited with exit code 1
